src/main/java/com/denizenscript/ddiscordbot/DiscordCommand.java

```diff
@@ -45,9 +56,33 @@
     // Use to message a Discord channel.
     // - discord id:mybot message channel:<discord[mybot].server[Denizen].channel[bot-spam]> "Hello world!"
 
+    // @Usage
+    // Use to add a role on a user in a Discord guild.
+    // - discord id:mybot role add user:<def[user_id]> role:324782895 guild:<def[guild_id]>
+
+    // @Usage
+    // Use to remove a role on a user in a Discord guild.
+    // - discord id:mybot role remove user:<def[user_id]> role:435345436457656 guild:<def[guild_id]>
+
+    // @Usage
+    // Use to set the status of the bot.
+    // - discord id:mybot status "Minecraft" "status:ONLINE" "activity:PLAYING"

```

Comment:
What are the valid options for these? Please update the `Description` and `Syntax` lines of the meta docs